### PR TITLE
fixing issue #681

### PR DESCRIPTION
**Pull request by maksimbormot**
Fixes https://github.com/marmelab/ng-admin/issues/681

### CODE REVIEW

**Comment by fzaninotto** [comment]
Please don't include build files, we'll do that after merging.

File: src/javascripts/ng-admin/Crud/list/maDatagridInfinitePagination.js [hunk 1]
```diff
@@ -17,19 +17,29 @@ export default function maDatagridInfinitePagination($window, $document) {
...
-                page = 1;
+                page = 1,
+                processing = false;
             function handler() {
-                if (body.offsetHeight - $window.innerHeight - $window.scrollY < offset) {
-                    if (page >= nbPages) {
-                        return;
+                if(processing) return;
+                processing = true;
+                var timer = setInterval(function(){
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/list/maDatagridInfinitePagination.js]: why do you need a `setInterval()` while the handler is already called on each scroll?

Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/list/maDatagridInfinitePagination.js]: this interval is never cleared, this is a memory leak

File: src/sass/sb-admin.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@ body {
 
 #page-wrapper {
     padding: 0 15px;
-    min-height: 568px;
+    //min-height: 568px;
```
Comment (fzaninotto) [inline src/sass/sb-admin.scss]: commented out code

**Comment by Kmaschta** [comment]
This PR is superseded by #1322, so I close this one.

Thanks for contributing! I reused your solution.

**Pull request closed without merging**